### PR TITLE
add old version of sdg scripts

### PR DESCRIPTION
**Pull request by n-h-diaz**

### CODE REVIEW

File: scripts/un/sdg/sdmx/cities.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+def get_cities(json, api_key):
+    '''Applies find entities API for given json.
+    Args:
+        json: Input json.
+        api_key: API key.
+    Returns:
+        API response.
+    '''
+    return requests.post(
+        'https://autopush.api.datacommons.org/v1/bulk/find/entities',
```
Comment (dwnoble) [inline scripts/un/sdg/sdmx/cities.py:41]: Should this use api.datacommons.org so UN can run it?

File: scripts/un/sdg/README.md [hunk 2]
```diff
@@ -39,4 +40,15 @@ python3 -m unittest discover -v -s ../ -p "*_test.py"
 ```
 
 Notes:
-* We currently drop certain series and variables (refer to `util.py` for the list) which have been identified by UN as potentially containing outliers.
\ No newline at end of file
+* We currently drop certain series and variables (refer to `util.py` for the list) which have been identified by UN as potentially containing outliers.
+
+### SDMX
+
+As reference, we provide an earlier version of the import scripts that utilized the UN API (which uses SDMX) in the `sdmx/` folder. Please note that these scripts may have errors and do not use the most up-to-date schema format, so should only be used as an illustration of the SDMX -> MCF mapping and should not actually be run.
```
Comment (juliawu) [inline scripts/un/sdg/README.md:47]: Suggestion: Bold "should not be run" or wrap the warning in a quote callout

**Review by dwnoble — APPROVED** [review]

File: scripts/un/sdg/sdmx/m49.csv [hunk 3]
```diff
@@ -0,0 +1,250 @@
+Country or Area	M49 code	ISO-alpha3 code
+Afghanistan	004	AFG
+Åland Islands	248	ALA
+Albania	008	ALB
+Algeria	012	DZA
```
Comment (juliawu) [inline scripts/un/sdg/sdmx/m49.csv:5]: Just want to confirm the format of this file is correct. I see it's a .csv but not comma-delimited / not evenly spaced

File: scripts/un/sdg/README.md (hunk 2 above)
Comment (juliawu) [reply scripts/un/sdg/README.md:47]: Also, consider adding "This script does not use the most up-to-date schema format. It should only be used as an illustration of the SDMX -> MCF mapping. Do not actually run" to the top of the relevant .py script files.

**Review by juliawu — APPROVED** [review]
Thanks! Just a few suggestions, but this lgtm.

File: scripts/un/sdg/sdmx/cities.py (hunk 1 above)
Comment (juliawu) [reply scripts/un/sdg/sdmx/cities.py:41]: +1

Comment (n-h-diaz) [reply scripts/un/sdg/sdmx/cities.py]: done

File: scripts/un/sdg/README.md (hunk 2 above)
Comment (n-h-diaz) [reply scripts/un/sdg/README.md]: done

File: scripts/un/sdg/sdmx/m49.csv (hunk 3 above)
Comment (n-h-diaz) [reply scripts/un/sdg/sdmx/m49.csv]: sorry changed to tsv

**Pull request merged**